src/zkbob/utils/ZkBobAccounting.sol

```diff
@@ -325,6 +345,21 @@ contract ZkBobAccounting {
         emit UpdateLimits(_tier, tl);
     }
 
+    // Tier is set as per the KYC Providers Manager recomendation only in the case if no
+    // specific tier assigned to the user
+    function _adjustConfiguredTierForUser(address _user, uint8 _configuredTier) internal view returns (uint8) {
+        uint8 tier = _configuredTier;
+        if (tier == 0) {
+            if (address(kycProvidersManager) != address(0)) {
+                (bool kycPassed, uint8 tmp_tier) = kycProvidersManager.getIfKYCpassedAndTier(_user);
+                if (kycPassed && tiers[tmp_tier].limits.tvlCap > 0) {
```

Comment:
If owner controls both: pool contract and manager contract, then such situation (when tier is not set) is not really possible. I don't think it then makes sense to make an extra SLOAD on each pool access.

Reply:
> If owner controls both: pool contract and manager contract, hen such situation (when tier is not set) is not really possible

I would agree but personally I don't like the fact that possibility of such situation is being increased with time: members of multisig could rotate and forget about sanity checks, a DAO could be inconsistent in the configuration changes. Moreover, we already do questionable SLOADs and SSTOREs to operate with `maxWeekly*` for each pool access. So a consistency check does not look excessive next to them.